flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/NetworkBufferCalculationTest.java

```diff
@@ -39,78 +33,66 @@
 public class NetworkBufferCalculationTest extends TestLogger {
 
 	/**
-	 * Test for {@link TaskManagerServices#calculateNetworkBufferMemory(TaskManagerServicesConfiguration, long)}
-	 * using the same (manual) test cases as in {@link TaskManagerServicesTest#calculateHeapSizeMB()}.
+	 * Test for {@link NetworkEnvironmentConfiguration#calculateNewNetworkBufferMemory(Configuration, long)}
+	 * using the same (manual) test cases as in {@link NetworkEnvironmentConfigurationTest#calculateHeapSizeMB()}.
 	 */
 	@Test
-	public void calculateNetworkBufFromHeapSize() throws Exception {
-		TaskManagerServicesConfiguration tmConfig;
+	public void calculateNetworkBufFromHeapSize() {
+		Configuration config;
 
-		tmConfig = getTmConfig(Long.valueOf(TaskManagerOptions.MANAGED_MEMORY_SIZE.defaultValue()),
+		config = getConfig(
+			Long.valueOf(TaskManagerOptions.MANAGED_MEMORY_SIZE.defaultValue()),
 			TaskManagerOptions.MANAGED_MEMORY_FRACTION.defaultValue(),
-			0.1f, 60L << 20, 1L << 30, MemoryType.HEAP);
+			0.1f, 60L << 20, 1L << 30, false);
```

Comment:
MemoryType.HEAP/OFF_HEAP looked a bit more readable.
could we use `memType  == MemoryType.OFF_HEAP` in `getConfig(.., memType)`?